app/Http/Controllers/Legacy/Web/CampaignsController.php

```diff
@@ -76,7 +94,24 @@ public function show(Campaign $campaign)
      */
     public function edit(Campaign $campaign)
     {
-        return view('campaign-ids.edit')->with('campaign', $campaign);
+        $causes = [
+            'No Cause',
+            'Animals',
+            'Bullying',
+            'Disasters',
+            'Discrimination',
+            'Education',
+            'Environment',
+            'Homelessness',
+            'Mental Health',
+            'Physical Health',
+            'Poverty',
+            'Relationships',
+            'Sex',
+            'Violence',
+        ];
+
+        return view('campaign-ids.edit')->with('campaign', $campaign)->with('causes', $causes);
```

Comment:
Might want to go with [the first option here](https://laravel.com/docs/5.5/views#passing-data-to-views) to pass an array instead of having two `with`s